lib/client/indexerClient.ts

```diff
@@ -0,0 +1,9 @@
+import { AutocratProgram, Dao, ProgramVersion } from "../types";
+import { MetaDAOClient } from "./client";
+
+export class MetaDAOIndexerClient implements MetaDAOClient {
+  async fetchDao(daoAddress: string): Promise<Dao | undefined> {
+    //TODO: use the indexer
```

Comment:
We can fetch known addresses (from github versions) but the DAOs are either derived from seed, or they will be able to be fetched from `getProgramAccounts`. Which we'll need to stub either way as the indexer will likely end up using this.

Reply:
https://github.com/metaDAOproject/meta-dao-frontend/pull/64/files#diff-75568db1341ff22eb9f152d0268bf30c0c7eb2c86a9fba566b44c04e4e0e5507R73

Reply:
These are the two ways we handle it, TECHNICALLY all can be handled with the program.dao.all() method assuming you have the program built and loaded.

But we'll need to discuss this load pattern as gPA (getProgramAccounts) is intense.

Reply:
I guess this is in regards to the rpcClient? Of course the indexer will need to initially get it's data this way, and I think we will probably use that pathway for that as well.

Reply:
Yea it's just that we have fetchDAO() with some method to point which we should use (or just have two different methods, but I do like the failover contained in the method).

So you have fetchDAO but technically we'd have already fetched this, we just need to load the data for that dao but if the API fails, the RPC should call to load. That's all I was thinking at this point is HOW.

Reply:
I see yeah if we already fetched all the DAOs and are just filtering, like when we fetch again we shouldn't have to call this again basically? Even probably in RPC client implementation.

Reply:
After discussion it was decided we will fetch all and then UI layer can cache with react-query. Resolving.